### PR TITLE
Add RESIZE operators to Realy TFLite frontend

### PR DESCRIPTION
**Pull request by apivovarov**
Add RESIZE operators to Realy TFLite frontend

Request to support `RESIZE_BILLINEAR` operators comes from https://github.com/dmlc/tvm/issues/3189

### CODE REVIEW

**Comment by apivovarov** [comment]
@FrozenGene @yongwww Could you have a look?

File: python/tvm/relay/frontend/tflite.py [hunk 1]
```diff
@@ -224,6 +226,53 @@ def convert_reshape(self, op):
...
+        else:
+            assert op.BuiltinOptionsType() == BuiltinOptions.ResizeNearestNeighborOptions
+            resize_options = ResizeNearestNeighborOptions()
+
+        op_options = op.BuiltinOptions()
+        resize_options.Init(op_options.Bytes, op_options.Pos)
+        align_corners = resize_options.AlignCorners()
+
+        # Use layout NHWC
+        out = _op.image.resize(in_expr, size, "NHWC", method, align_corners)
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: After previous, we could use `tuple(target_size)`

File: python/tvm/relay/frontend/tflite.py [hunk 2]
```diff
@@ -224,6 +226,53 @@ def convert_reshape(self, op):
...
+        assert len(input_tensors) == 2, "input tensors length should be 2"
+
+        # images, 4-D Tensor with shape NHWC.
+        input_tensor = input_tensors[0]
+        in_expr = self.get_expr(input_tensor.tensor_idx)
+
+        # size - 1-D int32 Tensor of 2 elements: new_height, new_width
+        size = self.get_tensor_value(input_tensors[1])
+        # convert numpy array to tuple
+        size = (size[0], size[1])
```
Comment (FrozenGene) [inline python/tvm/relay/frontend/tflite.py]: suggest better name. Like this:
``` python
target_size = self.get_tensor_value(input_tensors[1])
```

File: python/tvm/relay/frontend/tflite.py (hunk 1 above)
Comment (apivovarov) [reply python/tvm/relay/frontend/tflite.py]: fixed

File: python/tvm/relay/frontend/tflite.py (hunk 2 above)
Comment (apivovarov) [reply python/tvm/relay/frontend/tflite.py]: fixed

**Comment by apivovarov** [comment]
Frontend GPU test group failed with error
```
Traceback (most recent call last):

  File "/workspace/python/tvm/relay/frontend/tflite.py", line 235, in _convert_resize

    from tflite.ResizeNearestNeighborOptions import ResizeNearestNeighborOptions

ModuleNotFoundError: No module named 'tflite.ResizeNearestNeighborOptions'
```

It means that Jenkins worker uses old schema from Tensorflow 1.12
I updated `ubuntu_install_tflite.sh` to use schema from version 1.13
https://github.com/dmlc/tvm/blob/master/docker/install/ubuntu_install_tflite.sh#L38
merged 16 hours ago

# Schema v 1.13
https://raw.githubusercontent.com/tensorflow/tensorflow/r1.13/tensorflow/lite/schema/schema.fbs

**Comment by apivovarov** [comment]
I opened an issue for the build error https://github.com/dmlc/tvm/issues/3373

**Comment by apivovarov** [comment]
Fixed the PR to support both tflite versions - 1.12 and 1.13

**Comment by apivovarov** [comment]
@kevinthesun Can you merge it?

**Pull request merged**